index.md

```diff
@@ -159,6 +159,14 @@ We appreciate these partnerships and look forward to working with you all!
 ### **ESIIL Interns** 
 (please add your profiles)
 
+ <img style="float: right;" src="https://github.com/KiStroh/Kistroh-esiil-stars-webpage/raw/main/SELFIE_30.jpg"
+ width="125"/>
+ 
+### **Kianna Stroh**
+ * Sophmore at UTTC - Environmental Science and Research
+ * <a href = "mailto: stroh.kianna@stu.uttc.edu" target="_blank">**EMAIL** </a>
+ * <a href = "https://KiStroh.github.io/KiStroh-esiil-stars-webpage" target="_blank">**WEBSITE** </a>
```

Comment:
@KiStroh this link is broken - looks like it should be https://KiStroh.github.io/Kistroh-esiil-stars-webpage with the lowercase s. Please resubmit!